### PR TITLE
DOC: tidy 1.1.5 release notes

### PR DESCRIPTION
**Pull request by simonjayhawkins**

### CODE REVIEW

File: doc/source/whatsnew/v1.1.5.rst [hunk 1]
```diff
@@ -14,22 +14,21 @@ including other versions of pandas.
...
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
-- Regression in addition of a timedelta-like scalar to a :class:`DatetimeIndex` raising incorrectly (:issue:`37295`)
+- Fixed regression in addition of a timedelta-like scalar to a :class:`DatetimeIndex` raising incorrectly (:issue:`37295`)
 - Fixed regression in :meth:`Series.groupby` raising when the :class:`Index` of the :class:`Series` had a tuple as its name (:issue:`37755`)
 - Fixed regression in :meth:`DataFrame.loc` and :meth:`Series.loc` for ``__setitem__`` when one-dimensional tuple was given to select from :class:`MultiIndex` (:issue:`37711`)
 - Fixed regression in inplace operations on :class:`Series` with ``ExtensionDtype`` with NumPy dtyped operand (:issue:`37910`)
+- Fixed regression in metadata propagation for ``groupby`` iterator (:issue:`37343`)
+- Fixed regression in indexing on a :class:`Series` with ``CategoricalDtype`` after unpickling (:issue:`37631`)
+- Fixed regression in :class:`RollingGroupby` with the resulting :class:`MultiIndex` when grouping by a label that is in the index (:issue:`37641`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v1.1.5.rst:23]:
```suggestion
- Fixed regression in ``df.groupby(..).rolling(..)`` with the resulting :class:`MultiIndex` when grouping by a label that is in the index (:issue:`37641`)
```

(we don't have API pages for this, and people also don't necessarily know what `RollingGroupby` is)

**Review by jorisvandenbossche — COMMENTED** [review]
Looks good, small comment

File: doc/source/whatsnew/v1.1.5.rst (hunk 1 above)
Comment (simonjayhawkins) [reply doc/source/whatsnew/v1.1.5.rst:23]: sure. this pass was moreless just a move. no problem with committing this suggestion. but a second pass is ok too.

**Comment by simonjayhawkins** [comment]
test failure unrelated

```
=========================== short test summary info ============================
FAILED pandas/tests/plotting/frame/test_frame.py::TestDataFramePlots::test_area_lim
= 1 failed, 143863 passed, 3150 skipped, 1421 xfailed, 31 warnings in 2993.64s (0:49:53) =
```

**Pull request merged**